dbt_semantic_interfaces/dbt_semantic_interfaces/objects/aggregation_type.py

```diff
@@ -0,0 +1,17 @@
+from metricflow.object_utils import ExtendedEnum
+
+
+class AggregationType(ExtendedEnum):
```

Comment:
I think we can make this extend Python's builtin Enum. The ExtendedEnum doesn't do that much for us.

If we do need it, we can put it in the interfaces package, but I suspect we will not.